index.html

```diff
@@ -1,116 +1,229 @@
+            <p class="text-[12px]" id="project-text">
+              Listen to the speakers from various countries about the messages
+              of sharing and opening..
+            </p>
+          </div>
+          <div
+            class="flex bg-[#5f5f5f50] p-[20px] md:flex-col flex-row gap-4 justify-center items-center"
+          >
+            <img src="assets/program_icon_04.png" alt="white-board-logo" />
+            <h2 class="text-[20px] text-[#ec5242]">CC Exhibition</h2>
+            <p class="text-[12px]" id="project-text">
+              Appreciate various creations applying CC license of artists,
+              organized from Art Center Nabi.
+            </p>
+          </div>
+          <div
+            class="flex bg-[#5f5f5f50] p-[20px] md:flex-col flex-row gap-4 justify-center items-center"
+          >
+            <img src="assets/program_icon_05.png" alt="forum-logo" />
+            <h2 class="text-[20px] text-[#ec5242]">Forum</h2>
+            <p class="text-[12px]" id="project-text">
+              Have the time to share your thoughts and opinions with experts for
+              each topic.
+            </p>
+          </div>
+          <div
+            class="flex bg-[#5f5f5f50] p-[20px] md:flex-col flex-row gap-4 justify-center items-center"
+          >
+            <img
+              class="justify-center"
+              src="assets/program_icon_01.png"
+              alt="desk-logo"
+            />
+            <h2 class="text-[20px] text-[#ec5242]">Workshop</h2>
+            <p class="text-[12px]" id="project-text">
+              Try creating your own work using open source license rather than
+              just watching at it..
+            </p>
+          </div>
+          <div
+            class="flex bg-[#5f5f5f50] p-[20px] md:flex-col flex-row gap-4 justify-center items-center"
+          >
+            <img src="assets/program_icon_01.png" alt="desk-logo" />
+            <h2 class="text-[20px] text-[#ec5242]">CC Ignite</h2>
+            <p class="text-[12px]" id="project-text">
+              Get opportunities to network with CC affiliates around the world,
+              also after the summit..
+            </p>
+          </div>
+        </div>
+        <div class="md:hidden text-white bg-[#ec5242] px-[20%] py-[5%]">
+          <a href="">Join The Grape Escape 2023</a>
+        </div>
+        <div class="hidden md:flex flex-col items-center underline">
+          <a href="">
+            <p>SEE THE WHOLE PROGRAM</p>
+          </a>
+        </div>
+      </section>
+      <section class="px-[12%] flex flex-col items-center py-[3%]">
+        <h2 class="text-center text-black text-[20px]">Featured speakers</h2>
+        <hr class="w-[35px] bg-[#ec5242] h-[4px]" />
+        <div
+          id="speakers"
+          class="ml-[3%] mt-[72px] grid gap-[12px] grid-cols-1 md:grid-cols-2 justify-items-center"
+        ></div>
+      </section>
```

Comment:
**Featured speakers** section is broken in the mobile view. Kindly fix this issue. ⬇️ 
 I have recorded [this video](https://www.loom.com/share/995ea0b1c5eb4346a5f0d096bb8e3b5b) where you can verify the problem and also talked about the possible fixes.